gov_uk_dashboards/components/plotly/navbar.py

```diff
@@ -1,27 +1,23 @@
 """navbar"""
 
-from dash import html, dcc
+from dash import html
 
 
 def navbar(links):
     """A navigation bar for switching between dashboards."""
     return html.Nav(
-        html.Div(
-            html.Ul(
-                links, className="moj-side-navigation__list", id="navigation-items"
-            ),
-            className="moj-side-navigation govuk-!-padding-right-2",
-            role="navigation",
-        ),
-        className="dashboard-menu",
+        html.Ul(links, className="moj-primary-navigation__list", id="navigation-items"),
```

Comment:
does moj stand for Ministry of Justice? and if so can we name them something different :)

Reply:
It does but thats because we grabbed the CSS already from them. They will be another piece that will provide the scss file to fix all this